clients/gtest/matmul_gtest.yaml

```diff
@@ -917,4 +917,25 @@ Tests:
   bias_type: [f32_r]
   unit_check: 1
   gpu_arch: '94?'
+
+- name: matmul_gemm_mix_precisions_fp8
+  category: pre_checkin
+  function:
+    matmul: *real_mix_precisions_fp8
+  M: [127, 129]
+  N: [127, 129]
+  K: [127, 129]
+  transA_transB: *transA_transB_range
+  alpha: 1
+  beta: [ 0.0, 2.0 ]
+  scaleA: [1]
+  scaleB: [1]
```

Comment:
Please also test no scaleA(Or B,C,D case) in supported kernel

Reply:
Fixed